config.yml.dist

```diff
@@ -137,7 +137,8 @@ jwt:
         file: public-auth0.pem
     validation:
       iss: https://publiq-acc.eu.auth0.com/
-
+    domain: publiq-acc.eu.auth0.com
+    
```

Comment:
@konrad126 can you create an appconfig PR for this please? https://bitbucket.org/cultuurnet/appconfig/

As well as a PR in https://bitbucket.org/cultuurnet/udb3-vagrant/src for local installations.
You can find an example here: https://bitbucket.org/cultuurnet/udb3-vagrant/pull-requests/75